packages/features/ee/workflows/api/scheduleSMSReminders.ts

```diff
@@ -6,6 +6,7 @@ import dayjs from "@calcom/dayjs";
 import { defaultHandler } from "@calcom/lib/server";
 import prisma from "@calcom/prisma";
 
+import { noAlphanumericSenderIdSupport } from "../lib/aphanumericSenderIdSupport";
```

Comment:
```suggestion
import { noAlphanumericSenderIdSupport } from "../lib/alphanumericSenderIdSupport";
```
Found a misspelling in this import